bundles/net.tourbook/src/net/tourbook/ui/views/TourPausesView.java

```diff
@@ -126,19 +130,19 @@ public class TourPausesView extends ViewPart implements ITourProvider, ITourView
 
    private Menu      _tableContextMenu;
 
-   public class DevicePause {
+   private class DevicePause {
 
-      long type;
+      private long type;
 
-      long relativeStartTime;
-      long relativeEndTime;
+      private long relativeStartTime;
+      private long relativeEndTime;
 
-      int  serieIndex;
+      private int  serieIndex;
```

Comment:
private fields should start with an `_` https://github.com/mytourbook/mytourbook/discussions/919#discussion-4349284-permalink

Reply:
I'm not fully happy with these private fields. Normally, private field should not be accessible from outside of the class, but they are. So the original implementation with no private fields was more according to the source code rules.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/ea05dfd8-07ad-4a54-b034-bbdab64efa69)

Reply:
This is the [standard practice](https://rules.sonarsource.com/java/RSPEC-1104/) (**Class variable fields should not have public accessibility**) and a rule that makes sense to me.

Reply:
I don't want to declare all class fields as private and create for each a getter and setter method. I decide, if a field is private or not, depending on the current context.